Assets/MRTK/Tests/PlayModeTests/Experimental/BoundsControlTests.cs

```diff
@@ -268,6 +269,41 @@ public IEnumerator ScaleViaNearInteraction()
             yield return null;
         }
 
+        /// <summary>
+        /// Uses near interaction to scale the bounds control by directly grabbing corner - precise scaling 
+        /// </summary>
+        [UnityTest]
+        public IEnumerator ScaleNonUniform()
+        {
+            BoundsControl boundsControl = InstantiateSceneAndDefaultBoundsControl();
+            boundsControl.ScaleHandlesConfig.ScaleBehavior = HandleScaleMode.NonUniform;
+            yield return VerifyInitialBoundsCorrect(boundsControl);
+            var inputSimulationService = PlayModeTestUtilities.GetInputSimulationService();
```

Comment:
Is inputSimulationService being used somewhere?

Reply:
actually not anymore - i changed usage from playmodetestutilities to test hand. can be removed!